docker/reqmgr2ms-unmerged/monitor.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
the same argument here, bash may not be required and simpler /bin/sh can be used. Doing this way bash may not need to be installed in an image at all.

Reply:
same reply as above